go/app/main.go

```diff
@@ -28,14 +43,113 @@ func root(c echo.Context) error {
 func addItem(c echo.Context) error {
 	// Get form data
 	name := c.FormValue("name")
+	category := c.FormValue("category")
+	image := c.FormValue("image")
+
 	c.Logger().Infof("Receive item: %s", name)
+	c.Logger().Infof("Receive category: %s", category)
+	c.Logger().Infof("Receive image: %s", image)
+
+	hash := sha256.New()
+	// Write the image data to the hash function
+	hash.Write([]byte(strings.Split(image, ".")[0]))
```

Comment:
You did a nice job to learn how to call the hash function 👍 . 

But there's one key problem 😮 .
`image` contains the file name for the image, not the content of the image file.

So we need to read the file content and then send it to the `hash.Write` function.